pluto-plugins/plugins/network/core/lib/src/main/java/com/pluto/plugins/network/intercept/NetworkData.kt

```diff
@@ -11,42 +12,80 @@ class NetworkData {
         val method: String,
         val body: Body?,
         val headers: Map<String, String?>,
-        val sentTimestamp: Long
+        val sentTimestamp: Long,
     ) {
+        data class GraphqlData(
+            val queryType: String,
+            val queryName: String,
+        )
+
+        val graphqlData: GraphqlData? = parseGraphqlData()
+
+        private fun parseGraphqlData(): GraphqlData? {
+            if (method != "POST" ||
+                body == null ||
+                !body.isLikelyJson
```

Comment:
Maybe more optimized way is to check header "x-apollo-operation-name" but it is not the part of the graphql specification and is provided only by apollo library.